DQM/SiPixelPhase1Heterogeneous/python/SiPixelPhase1HeterogenousDQM_FirstStep_cff.py

```diff
@@ -35,13 +36,25 @@
   topFolderName = 'SiPixelHeterogeneous/PixelVertexSoAGPU',
 )
 
+siPixelPhase1MonitorRecHitsSoACPU = siPixelPhase1MonitorRecHitsSoA.clone(
```

Comment:
isn't this conflicting with https://github.com/cms-sw/cmssw/pull/37798/files#diff-9028f77f088f106cd424e7fa0b3ee30e8f1efd6f2f489b3b8ee3fde183babfebR37 ?